R/stageData.R

```diff
@@ -53,11 +53,13 @@ getStaged <- function(rec) {
         message("Skipping download (zip file located on disk) ...")
       }
       utils::unzip(tempzip, exdir = gisTempdir)
-      if (rec$download_data_standard == 'shp') {
+      if (rec$download_data_standard %in% list('shp','gdb')) {
         return(sf::st_read(file.path(gisTempdir, rec$download_filename)))
```

Comment:
yes, but there is a caveat. If the database name and the feature class name are the same this works, if there are more than one feature class or the feature class name is different, you must specific the feature class to read from the database. For the CDC data, there is one feature class in the database with the same name as the database -- it works. This may not be the case for all GDB's we encounter. I will add a TODO comment above the code with the same explanation (next pull request).